### ISSUE TITLE
Tag v1.3.0 not found in repository docker.io/openshift/origin

### ISSUE THREAD

**Issue by harobed** [openshift/openshift-ansible#1819]
« Tag v1.3.0 not found in repository docker.io/openshift/origin » on "prod" branch:

```
TASK: [openshift_cli | Pull CLI Image] ****************************************
failed: [atomic-test-master-1] => {"changed": true, "cmd": ["docker", "pull", "openshift/origin:v1.3.0"], "delta": "0:00:03.145601", "end": "2016-04-27 15:07:16.204415", "rc": 1, "start": "2016-04-27 15:07:13.058814", "warnings": []}
stderr: Tag v1.3.0 not found in repository docker.io/openshift/origin
stdout: Trying to pull repository docker.io/openshift/origin ... not found

FATAL: all hosts have already failed -- aborting
```

**Comment by sdodson** [comment]
The Prod branch is used for internal release automation processes. Please use master and let me know if the problem still occurs, thanks.

**Comment by sdodson** [comment]
Should be fixed by #1803

**Comment by harobed** [comment]
Ok, I try master branch, but now I'm blocked by #1820
